### DISCUSSION TITLE
HCIBench 2.8.3

### DISCUSSION THREAD

**Discussion by cwei44** [vmware-labs/hci-benchmark-appliance#7] [Show and tell]
1. Removed vSAN Observer
2. Modification on vSAN Debug Mode
3. Support vNVMe controller deployment and testing at code level
4. Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/vmware-labs/hci-benchmark-appliance/releases/tag/2.8.3'>HCIBench 2.8.3</a>.</em>

**Comment by shadarzoll** [comment]
Hi, 
first of all, many thanks for this great testing suite and the time and effort which have been put into it. 

We have downloaded the 2.8.3 version and run into the issue that the deployed worker VMs boot disk is empty (The vmdk file is 0 bytes in size). Is there anyway we can troubleshoot this? We are running Vcenter 7U3. 
We also tested the 2.8.2 version with the same result. 
We can see the worker VMs getting deployed via a OVF deployment. In the test scenario all the test vmdks get attached. But in both, the "real" test and the validation the boot disk is always 0 byte in size. 
In the HCI logs we only see that the worker VMs are not reachable via their IPs, which is expected as they can't boot from the 0 byte boot disk. 
We already found out that port 443 on the ESXi was not reachable from the HCI Bench VM, but that is working now. 
Is there anything else we can check?
Is there an option to increase the log level?
Are there any logs somewhere from the OVF deployment?
We don't have a vsan or other HCI in place. We just wanted to use this to get some perf data from our SAN. Not sure if that makes a difference.

If this is not the correct place to ask this question please let me know where I should post it.
I didn't wanted to open an issue as no ones seems to have this issue and the tool is working for fine for everyone.

Many thanks

Sven

**Reply by var1amov** [reply]
+1 for verbose logging. Text like "NoPermission" is great point to start, but a lot of time could be saved if it will contain details

**Comment by DowneyOu** [comment]
Hi, @cwei44 

I have problem to access the grafana link in http"//my_hcibench_IP:3000

There are all good at all testing progress (deploy vm , test io , see report in pdf file).

However , I cannot access the port 3000 grafana page like following link
http://my_hcibench_IP:3000/d/fio/hcibench-fio-monitoring?orgId=1&var-Testname=easy-run-1724803781

Can anyone give me the clue how to check it ?


( BTW , I can access http://my_hcibench_IP:8443 without any problem and there is no ACL in my network)

**Comment by PavelKovar78** [comment]
Hi,
in our environment we are using 172.17.0.0/16 network range. I can see this network is allocated to docker0 inside HCIBench appliance.
Is it possible to change this docker network to something different? Can I change it by modifying docker configuration and where?
Thanks!

**Comment by julienhokini** [comment]
Hi, i have deploy HCIBench 2.8.3 , my validation test is ok. But all graph is empty in grafana, no data in all graph. I have deploy on last vcenter 8 version.
